test/functional/test_framework/messages.py

```diff
@@ -893,7 +889,6 @@ def __init__(self):
         self.nTransactions = 0
         self.vHash = []
         self.vBits = []
-        self.fBad = False
```

Comment:
even though this is unused, is it really a good idea to remove code that initializes default values?

Reply:
@jb55 Do you mean the fact that `fBad` was still in `__slots__`? Now it is gone so `fBad` is history. Updated version looks good? :-)

Reply:
perhaps, 3a4449e9ad945313c6637283757de8d539cf790f states:

> Classes use __slots__ to ensure extraneous attributes aren't accidentally added by tests, compromising their intended effect.

this seems to be reversing the original intent of that commit. /cc @JustinTArthur ?

Reply:
If `fBad` isn't going to be used, removing it won't go against the intentions of the slots work, which is just there to protect test developers from making tests that don't do what the developer thinks it does.

I just want to make sure the removal doesn't break the intentions of those message module classes, which provide test developers with Python equivalents of the common Core data objects—`fBad` is still in the Core C++ classes. If we generally only include attributes we need for existing tests, then no problem removing it and someone can re-add it later if their test incorporates it. I don't have enough history with the functional test suites to make a judgement call.

Reply:
+1 remove `fBad`.

Reply:
+0 sounds good to me.

nit: it looks like the fBad slot removal was squashed into the wrong commit.

Reply:
@jb55 Thanks for letting me know about the squash mixup. Now fixed :-)